src/applications/personalization/profile-2/components/ProfileMobileSubNav.jsx

```diff
@@ -155,7 +171,13 @@ const ProfileMobileSubNav = ({
         lastMenuItem.current.addEventListener('keydown', overrideTab);
       } else {
         document.removeEventListener('keydown', closeOnEscape);
-        openMenuButton.current.focus();
+        // Only set the focus on the menu trigger button if the menu just
+        // closed. This is mostly done to prevent overriding the focus
+        // management that occurs in the hook that fires on first render, which
+        // sets the focus on the h1
+        if (wasMenuOpen) {
+          openMenuButton.current.focus();
```

Comment:
Previously this would set the focus on the button when the mobile menu was mounted which prevents us from setting the focus on the H1. With this change the focus is only set on the button if the menu has just closed, which is actually the intended behavior anyway.